fitnesse/resources/bootstrap-plus/csv/toolTipData.csv

```diff
@@ -0,0 +1,25 @@
+Press '?' for keyboard shortcuts.,
+Bullet lists are created by prefixing a space and an * to a line,
+Headers are created by prefixing a line with !1 or !2 or !3 or !4 or !5 or !6,
+Numbered lists are created by prefixing a space and a number to a line,
+Picture can be added by using the !img marker followed by a URL,
+Did you know there is a dark theme for the FitNesse toolchain?,
+Did you know there is an contexthelper?,
+Did you know you add non-wiki files to 'http://localhost:####/files/'?,
+Did you know you can add/edit templates when you go to FitNesse/templatelibrary?,
+Did you know you can press 'CTRL' + 'space' in the editor to use autocomplete?,
+Did you know you can rename/move/delete existing pages,
+Did you know you can run tests based on their tag? You can do this with 'Run test with tags'.,
+Did you know you can save your search as a link? This is found at the bottom of your search page.,
+Did you know you can add an excel table by copying it and then pasting it into the editor? After this push Spreadsheet to FitNesse and you're done.,
+Did you know you can style your syntax? For more information see in the full reference guide "MarkupLanguageReference".,
+Did you know you can turn on autosave and autovalidate in settings?,
+Did you notice that there are special page titles?,
+You can add a comment to the wiki markup text by simply putting the '#' sign as the first character of a line,
+You can add an closed collapsible section by writing '!*> title [enter] text *!',
+You can add an open collapsible section by writing '!* title [enter] text *!',
+You can give a page an expiration date with '-e <days>' in the editor. When passes the subject will be deleted.,
+You can make a table of contents with '!headings -' in the editor.,
+If you click 'Recent Changes' in the 'Tools' a history of changes to the page will be shown.
+If you click 'Versions' in the 'Tools' a list of old versions of the page will be shown.,
+If you click on 'Where Used' in the 'Tools' a list of every page that references that page will be shown.
```

Comment:
Nog een tooltip die nu nog mist volgens mij...
Did you know there is a dark mode?